src/forecast.py

```diff
@@ -66,6 +66,7 @@ def forecast_page():
             forecast_models.remove("National_xg")
             st.sidebar.warning("National_xg only available for National forecast.")
 
+    show_prob = st.sidebar.checkbox('Show Probabilities', value=False)
```

Comment:
I wonder if we could make this `Show Probabilistic Forecast` or `Show Probabilistic Range`

Reply:
It could also be good to either only have the checkbox show when `gsp_id = 0` or possibly add another warning like on `line 67` that says probabilistic data is currently only available for the National Forecast.

Reply:
yea nice one, Ill go for `gsp_id=0` for the moment